UnityProject/Assets/Scripts/Core/Lighting/LightSource.cs

```diff
@@ -306,11 +307,20 @@ private void TryRemoveBulb(HandApply interaction)
 		{
 			try
 			{
-				var handSlot = interaction.PerformerPlayerScript.DynamicItemStorage.GetActiveHandSlot();
+				var handSlots = interaction.PerformerPlayerScript.DynamicItemStorage.GetNamedItemSlots(NamedSlot.hands);
```

Comment:
I feel like getting the active hand slot was the way to go, what isn't working there? It isn't returning the gloves in the slot?

Reply:
active hand slot is the slot that you hold stuff in
not the slot that you use to hear gloves

Reply:
ok so that is what isn't working, they should be the same hand (the one that holds stuff and interacts and the one that wears gloves)

Reply:
active hands are `leftHand` and `rightHand` slots which don't house the item we want to check
`hands` is one that is used for customizations like `eyes`, `feet`, etc.

Reply:
Please don't resolve conversations like that, at least wait until the reviewer gives back a positive answer. If you're not willing to make the hands be hands both in interaction and in clothing slot, which is fair because it is a bit change, at least add a TODO comment stating it has to be done.

Reply:
but you wouldn't be able to interact with anything because your hands are taken up with gloves

Reply:
I mean, Bod is just repeating what I said now.

Reply:
Added the TODO and I've added a little note explaining the difference between the two currently.